tests/test_cli.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf8
+from __future__ import unicode_literals, print_function, division
```

Comment:
Nit: I would only add print_function and division if there is an acual use of them (so the former probably never in libary code)